Fixed package names

BenWertz opened the pull request:
I know that lowerCamelCase is the way we're *supposed* to do things, but it's not the way we *are* doing things, and if I change the folders I get some really bad conflicts, so I'm changing the package names back to what they were. In any case, the folders weren't renamed, so there was an error in every file I had to change, and now there isn't.

Seriously, in VSCode every file in AutonomousTools, Autonomous, and HatchCommands is all red. It needs fixing.

BenWertz committed:
Fixed package names

I know that lowerCamelCase is the way we're *supposed* to do things, but it's not the way we *are* doing things, and if I change the folders I get some really bad conflicts, so I'm changing the package names back to what they were. In any case, the folders weren't renamed, so there was an error in every file I had to change, and now there isn't.

122004 commented:
NATHANIEL! It's super helpful that you're fixing package names, (thank you!) but all affiliating areas of the code (i.e. folder names) also need to be changed...

BenWertz committed:
Moved HatchControls to be on the same level as all the other teleop commands.

It was in HatchControls, but ClimberControls, DriveControls, and CargoControls were on a different level in the file system. I fixed it, and fixed imports elsewhere, so everything still works.

122004 reviewed (approved):
👍

BenWertz committed:
Added in sensors, added in NetworkTables entries on collector and duck sensors for driver station.

I also updated WPILib again, so it can build.
